docker-compose.yml

```diff
@@ -5,8 +5,6 @@ services:
     build: .
     image: aeternity/aeternity:${IMAGE_TAG-latest}
     hostname: node1
-    environment:
-      EPOCH_CONFIG: /home/aeternity/aeternity.yaml
```

Comment:
How is user config discovery meant to work without this var?
I see in line 14 that config is mapped to `/home/aeternity/aeternity.yaml` though not considered in https://github.com/aeternity/aeternity/wiki/User-provided-configuration